velox/docs/functions/json.rst

```diff
@@ -2,6 +2,95 @@
 JSON Functions
 ==============
 
+JSON Format
+---------------
+JSON is a language-independent data format that represents data as
+human-readable text. A JSON text can represent a number, a boolean, a
+string, an array, an object, or a null, with slightly different grammar.
+For instance, a JSON text representing a string must escape all characters
+and enclose the string in double quotes, such as ``"123\n"``, whereas a JSON
+text representing a number does not need to, such as ``123``. A JSON text
+representing an array must enclose the array elements in square brackets,
+such as ``[1,2,3]``. More detailed grammar can be found on https://www.json.org.
+
+JSON Vectors
+---------------
+There are a number of Presto JSON functions expecting JSON-type inputs. An
+application of Velox that uses these functions may need to create JSON-type
+vectors. In the Velox internal implementation, the JSON type inherits the
+VARCHAR type. The manipulations of these vectors are hence similar. To create
+a JSON-type vector, one can use ``BaseVector::create(JSON(), size, pool)``
+that creates a vector of StringView.
```

Comment:
StringView -> StringViews

maybe clarify, `"a flat vector of StringViews, i.e. FlatVector<StringView>"`